### DISCUSSION TITLE
Window layout for 1.1

### DISCUSSION THREAD

**Discussion by CarlGao4** [CarlGao4/Demucs-Gui#35] [Polls]
I'm thinking of a new window layout using tabbed view for 1.1, so we can add more functions and options without crowding all those together which makes the window bigger and bigger.

||1.0|New look|
|-----|-----|-----|
|Load model (Windows)|![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/dcfb587c-2680-4536-964a-af8aa6dc5a2a)|![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/1298eb23-2573-4af2-9e29-f5d721bfbb41)|
|Load model (macOS)|![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/030338b0-1b3c-40da-9dd2-3646abdaca56)|![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/59aaf13d-0ac3-4c1f-a0ea-f445f69c2788)|
|Main window (Windows)|![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/fe6245e4-35dd-4fae-8247-b53f47e4b20c)|![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/a1887bf6-dc66-456d-99d7-70f0ec68c085)|
|Main window (macOS)|![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/cd21a096-f8ee-433d-a687-0049a1fec6f6)|![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/4b9b488d-65c3-4b17-9e9a-4ea3de215ecf)|

**Comment by CarlGao4** [comment]
![image](https://github.com/CarlGao4/Demucs-Gui/assets/57826826/f94d1ce9-6d6d-416f-b624-0a93345c3a53)

According to the vote results during past two weeks, I'll switch to the new UI layout in 1.1

**Comment by flightmansam** [comment]
This is me being a noob since I am very new to this project, but why even have a pane to select the model if it is locked out? Is there a way to change model after startup? If not might be better to have some status text somewhere showing the model and the configurations. Anyway, just an idea!

**Reply by CarlGao4** [reply]
I think it's better to show some parameters about the model

**Reply by flightmansam** [reply]
Just an idea! Trying it out for the first time last night (even with the 1.0.x) from a UX point of view it made it feel like the model is configurable retrospectively. Quitting the app sounds to change the model sounds like a core implementation thing, but a bit strange. Anyway I am rambling. Love the app, keep up the awesome work!

**Reply by CarlGao4** [reply]
I tried to allow users to switch models without restarting the app, but it seems that I can't find a method to completely free a model. Or, more memory will be required if you switch models.
